client/overview/task-list/index.js

```diff
@@ -31,24 +201,35 @@ const TaskList = ( { tasks } ) => {
 				</div>
 			</CardHeader>
 			<CardBody>
-				<CollapsibleList
-					collapsed={ false }
-					collapseLabel={ __( 'Hide tasks', 'woocommerce-payments' ) }
-					expandLabel={ __( 'Show tasks', 'woocommerce-payments' ) }
-				>
-					{ tasks.map( ( task ) => (
+				<ListComp animation="custom" { ...listProps }>
```

Comment:
I think we could make use of the `slide-right` animation here. Or copy the same CSS we use in WC Admin (that's why we use `custom` in WC Admin). It would be nice to use one or the other, as dismissing without animations feels weird.

Reply:
Good idea! I added the animation in the commit 49b8be4